std/regex/internal/parser.d

```diff
@@ -921,6 +921,7 @@ struct Parser(R)
                     op = Operator.Union;
                     goto case;
                 case ']':
+                    addWithFlags(set, last, re_flags);
                     set |= last;
```

Comment:
Mmm, wait a sec. This line adds a character as is whereas the one above it does the same but respecting casefolding. So just delete the redundant `set |= last;`